tools/myst/myst.edl

```diff
@@ -403,6 +403,8 @@ enclave
             gid_t host_egid   
         );
 
+        long myst_fdatasync_ocall(int fd);
```

Comment:
Can we also implement `fsync` for hostfs in this PR?

Reply:
Added.